flights/models.py

```diff
@@ -0,0 +1,65 @@
+from django.db  import models
+
+class Flight(models.Model):
+    departure_time = models.DateTimeField()
+    arrival_time   = models.DateTimeField()
+    flight_time    = models.DateTimeField()
+    seat           = models.ManyToManyField("Seat", through="Flight_seat")
```

Comment:
```suggestion
    seats           = models.ManyToManyField("Seat", through="Flight_seat")
```
M2M 관계의 데이터는 복수의 데이터를 나타내기 때문에 복수형으로 써주세요.